DigitalLearningSolutions.Data/DataServices/ActivityDataService.cs

```diff
@@ -47,14 +47,16 @@ public ActivityDataService(IDbConnection connection)
                         Registered,
                         Completed,
                         Evaluated
-                    FROM tActivityLog
+                    FROM tActivityLog AS al
+                    JOIN Applications AS ap ON ap.ApplicationID = al.ApplicationID

```

Comment:
When a relational table is used for the sole purpose of filter then an `exists/not exists` where clause would be the preffered convention I beleive the, rather that using a `JOIN`.